### PR TITLE
Dh 552 add a heading above the filters and make the headings consistent

### PR DESCRIPTION
**Pull request by DilwoarH**
**Changes**
- Changes wording on Keyword Search from "Keywords" to "Keyword Search".
- Adds in "Apply Filters" header for filters.

Ticket: https://trello.com/c/P7HpqQAU/552-add-a-heading-above-the-filters-and-make-the-headings-consistent

### CODE REVIEW

File: app/templates/search/_filters.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<h2>Apply Filters</h2>
```
Comment (samuelhwilliams) [inline app/templates/search/_filters.html]: It should be `Apply filters`; second word lowercase.

File: app/templates/search/_services_filters.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
   with
   id = "keywords",
   name = "q",
-  label = "Keywords",
+  label = "Keyword Search",
```
Comment (samuelhwilliams) [inline app/templates/search/_services_filters.html]: Same here, `Search` show be lowercase.

**Comment by DilwoarH** [comment]
@samuelhwilliams Requested changes have been made.

**Comment by samuelhwilliams** [comment]
5px still looks too close. Let's go for 10px; tried that out and it looks more consistent with the `Choose a category` bit. Then we can get this in.

**Comment by DilwoarH** [comment]
Changed to 10px.

**Pull request merged**